doc/design/test-dandisets-1.md

```diff
@@ -0,0 +1,111 @@
+
+### Hide draft-only dandisets
+
+Similarly to hiding explicitly marked test dandisets, we could, by default, also
+hide dandisets that have no published versions (as above, these dandisets would
+still appear for the owners). As the public may *want* to see draft-only
+dandisets, everyone would be able to toggle this mode with a setting.
+
+**NOTE: this feature has already been implemented.**
+
+### Hide dandisets with no assets
+
+Another variation on the previous proposal: hide dandisets that contain no
+assets. These dandisets are a subset of draft-only dandisets, since publishing a
+dandiset requires assets to be present.
+
+This option is problematic on its own--it is a valid use case to declare
+scientific intentions by creating a draft dandiset with no assets, then collect
+and add data only after months or years of real scientific work. That is to say,
+an assetless dandiset that nonetheless has a description, funding agent,
+contact person, owners, etc. may indeed be fully valid.
+
+**NOTE: this feature has already been implemented.**
+
+### Add questions to dandiset creation page
+
+Adding questions such as:
+- Do you welcome collaborators? (Y/N)
+- When do you anticipate posting data to the dandiset?
+
+may help to stem the tide of abandoned dandisets.
+
+The collaborators question, answered positively, would add a badge to the
+dandiset's landing page and search result display indicating the call for
+collaboration.
+
+The anticipation date question would delay considering the dandiset abandoned
+until after that date.
+
+### Enable deletion of dandisets
+
+Part of the problem may be that people simply cannot delete their own abanonded
+dandisets. Offering a delete button with the proper semantics could help here.
+
+The archive does not allow for deletion of *published* dandisets, by design. But
+dandisets that are draft-only should be deleteable. (Note that, in principle,
+admins *can* delete published versions, but this is intended only for extreme
+circumstances.)
+
+And, as an extension to that idea, a published dandiset should be able to be
+"sealed" by having its draft dandiset deleted, thus closing it off to further
+updates. This would serve as a signal that the dandiset is in its final state
+and has become a permanent member of the archive. Owners and admins will be able
+to "unseal" sealed dandisets in case further modification is needed.
+
+### Delisting of inactive dandisets
+
+Dandisets in a certain set of conditions will be automatically "delisted",
+meaning they will not appear in listings or searches other than to the owners
+themselves. Additionally, a delisted dandiset will become read-only so that only
+actively listed dandisets can be used by their owners to store data.
+
+Conditions for delisting can include the following:
+- draft-only and assetless for more than 7 days
+- draft-only and no activity for more than 14 days
+- published with no activity for more than 30 days (also see "sealing" above)
```

Comment:
we would refine these conditions, but the sentiment holds. i would add not just delisting, but deletion as well as a condition.